### PR TITLE
rsx/common/d3d12: Support non default alpha function

### PR DESCRIPTION
**Pull request by vlj**
Fix After burner climax cloud effects.

### CODE REVIEW

**Comment by tambry** [comment]
> Fix After burner climax butt effects

Butt effects?

File: rpcs3/Emu/RSX/GCM.cpp [hunk 1]
```diff
@@ -790,6 +790,22 @@ rsx::window_pixel_center rsx::to_window_pixel_center(u8 in)
 	throw EXCEPTION("Unknow window pixel center %x", in);
 }
 
+rsx::comparaison_function rsx::to_comparaison_function(u16 in)
```
Comment (tambry) [inline rpcs3/Emu/RSX/GCM.cpp]: Should be _comparison_.

**Comment by kd-11** [comment]
If the alpha threshold computation is correct, I should adapt it for GL; If i'm not mistaken, the calculation for the value is what's missing.

**Comment by vlj** [comment]
yes the GL backend uses the fixed function to handle alpha testing so the threshold is to be fixed.

**Pull request merged**